src/test/java/com/thomashan/coup/action/ActionValidatorUtilTest.java

```diff
@@ -49,7 +46,7 @@ public void testCheckIfActionTypeIsAllowable_NoException_IfActionTypeInAllowable
     }
 
     @Test
-    public void testCheckActionPlayerIsActive_ThrowsException_IfActionPlayerIsNotActive() {
+    public void testCheckActionPlayerIsActive_GivenPlayerIsNotActive_ThrowsException() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unit tests should not contain more than 1 assert(s).](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)